scripts/build-package.js

```diff
@@ -0,0 +1,36 @@
+const glob = require('glob');
+const path = require('path');
+const npmRun = require('npm-run');
+const yargs = require('yargs');
+
+/**
+ * Runs the build-babel or build-webpack npm target for the specified package.
+ * @param packageName Name of the package as listed in ./packages/
+ */
+function runBuildTask(packageName) {
+  const npmTask = packageName !== 'react-component-playground' ? 'build-babel' :
+      'build-webpack';
+  const stdout = npmRun.execSync(`PACKAGE=${packageName} npm run ${npmTask}`, {
+    cwd: __dirname,
+  });
+
+  /* eslint-disable no-console */
+  console.log(stdout.toString('utf-8'));
+}
+
+glob('./packages/*react-*', null, (err, files) => {
+  const allPackages = files.map(f => path.basename(f));
+  const argv = yargs
+      .usage('Usage: $0 <package>')
+      .help()
+      .argv;
+  const targetPackage = argv._[0];
+
+  if (!targetPackage) {
+    allPackages.forEach(pkg => (runBuildTask(pkg)));
```

Comment:
> A config file with the dependencies would be useful for this

At scale, I agree, but here are few things to consider:
1. The dependency graph doesn't matter between packages that are just Babel-compiled (all but the playground), because no bundling occurs and the dependencies are resolved later in the user's codebase
2. There is only one package that bundles and there aren't any plans to add another (the reason why CP bundles with webpack is its use of CSS)
3. This dep graph would still not be needed until we'd have a webpack-bundled package depend on another webpack-bundled package, which we can assume will never be the case

That said, I think we should go for the easy solution, even if that means another `if` for `react-component-playground` and dropping the list idea altogether. What would be useful is a comment leaving breadcrumbs for future generations, stating that this solution doesn't work when other packages start depending on each other to build (only one does now, see point 1.)